openhands/runtime/utils/bash.py

```diff
@@ -329,6 +329,6 @@ def run(self, action: CmdRunAction) -> CmdOutputObservation | FatalErrorObservat
                 interpreter_details=python_interpreter,
             )
         except UnicodeDecodeError as e:
-            return FatalErrorObservation(
-                f'Runtime bash execution failed: Command output could not be decoded as utf-8. {str(e)}'
+            return ErrorObservation(
```

Comment:
ditto - this shouldn't be an ErrorObservation (not an agent error), but rather an exception

Reply:
This one I'm not sure about--stdout can totally output things other than unicode. This one seems recoverable

Reply:
Let's keep this then! This was very uncommon irrc